code/MathSpamProtectorField.php

```diff
@@ -122,20 +131,23 @@ public static function get_math_question()
     }
 
     /**
-     * Checks the given answer if it matches the addition of the saved session 
+     * Checks the given answer if it matches the addition of the saved session
      * variables.
      *
      * Users can answer using words or digits.
      *
      * @return bool
      */
-    public static function correct_answer($answer)
+    public function correct_answer($answer)
```

Comment:
Similarly to `get_math_question`, could you please rename this to `isCorrectAnswer` maybe?